src/main/java/Task.java

```diff
@@ -0,0 +1,24 @@
+import java.util.*;
+
+public class Task {
+    public static final String descriptionPrefix = "description: ";
+    private String description;
+    private boolean important;
+    List<String> pastDescription = new ArrayList<>(); // a list of past descriptions
+
+    public Task(String d) {
+      this.description = d;
+      if (!d.isEmpty())
+          this.important = true;
+    }
+
+    public String getAsXML() { return "<task>"+description+"</task>"; }
```

Comment:
return string should be stored in a meaningful variable name. That variable should then be used.